### PR TITLE
changelog update

### PR DESCRIPTION
**Pull request by dmatetelki**
i'd like to tag it as 1.0.1

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,5 +1,17 @@
 This is a running log of changes to libvmod-memcached.
 
+libvmod-memcached 1.0.1 (2017-03-07)
+------------------------------------
+
+ * Small code formatting, travis and doc fixes.
```
Comment (dridi) [inline CHANGES.rst]: "Minor fixes" is enough

Comment (dridi) [reply CHANGES.rst]: Enough for the whole changelog except the RPM part.

**Comment by dridi** [comment]
Missing version bumps:

https://github.com/varnish/libvmod-memcached/blob/9f929ee1dcf4e533caf657a5e9ef620fa36cd756/configure.ac#L3

https://github.com/varnish/libvmod-memcached/blob/9f929ee1dcf4e533caf657a5e9ef620fa36cd756/README.rst#L13

**Comment by dridi** [comment]
Before doing a release, could you test #19 on our supported platforms and merge if it's all good?

**Comment by dmatetelki** [comment]
@Dridi : damn, i got this message after i tagged master :P

**Pull request merged**